diff --git a/modules/mod-coa-lottery/src/CoALottery.cpp b/modules/mod-coa-lottery/src/CoALottery.cpp
index bca93167..01a8055a 100644
--- a/modules/mod-coa-lottery/src/CoALottery.cpp
+++ b/modules/mod-coa-lottery/src/CoALottery.cpp
@@ -15,6 +15,7 @@
 #include "MailMgr.h"
 #include "Map.h"
 #include "MapMgr.h"
+#include "CellImpl.h"
 #include "CreatureTextMgr.h"
 #include "TemporarySummon.h"
 #include "TaskScheduler.h"
